apps/docusaurus/docs/reference/indexer.md

```diff
@@ -0,0 +1,389 @@
+| after_value                | String  | The value of a token property after the transaction. Ex. "100"                                                                           |
+| aptos_names_from           | Table   | Use the [Hasura explorer](#indexer-api-reference) to see these sub-fields.                                                               |
+| aptos_names_from_aggregate | Table   | Use the [Hasura explorer](#indexer-api-reference) to see these sub-fields.                                                               |
+| aptos_names_to             | Table   | Use the [Hasura explorer](#indexer-api-reference) to see these sub-fields.                                                               |
+| aptos_names_to_aggregate   | Table   | Use the [Hasura explorer](#indexer-api-reference) to see these sub-fields.                                                               |
+| before_value               | String  | The value of a token property before the transaction. Ex. "50"                                                                           |
+| current_token_data         | Table   | Use the [Hasura explorer](#indexer-api-reference) to see these sub-fields.                                                               |
+| entry_function_id_str      | String  | The identifier of the function called in this transaction. Ex. "transfer"                                                                |
+| event_account_address      | String  | This is an Aptos account address related to the event. Ex. "0x50bc83f01d48ab3b9c00048542332201ab9cbbea61bda5f48bf81dc506caa78a"          |
+| event_index                | bigint  | Index of the event within the transaction. Ex. 1                                                                                         |
+| from_address               | String  | This is an Aptos account address from which the token was sent. Ex. "0x50bc83f01d48ab3b9c00048542332201ab9cbbea61bda5f48bf81dc506caa78a" |
+| is_fungible_v2             | Boolean | Indicates whether the token is fungible. Ex. False for NFTs.                                                                             |
+| property_version_v1        | bigint  | The version of the token's properties under schema version 1. Ex. 1                                                                      |
+| to_address                 | String  | This is an Aptos account address to which the token was sent. Ex. "0x123abc456def7890abcdef1234567890abcdef1234"                         |
+| token_amount               | bigint  | The amount of the token transferred in this activity. Ex. 3                                                                              |
+| token_data_id              | bigint  | Unique identifier for this particular token's data. Ex. 123456789                                                                        |
```

Comment:
For token standard v1, this is derived from a combination of creator_address, collection_name, and token_name.